core/src/test/scala/unit/kafka/integration/TopicMetadataTest.scala

```diff
@@ -165,7 +165,7 @@ class TopicMetadataTest extends JUnit3Suite with ZooKeeperTestHarness {
                                 metadata.topicsMetadata.head.partitionsMetadata.nonEmpty)
                               metadata.topicsMetadata.head.partitionsMetadata.head.isr
                             else
-                              ""))
+                              ""), 6000L)
```

Comment:
To speed up the unit test, perhaps we can configure a smaller isrChangePropagateInterval?